src/Duke/ui/Ui.java

```diff
@@ -0,0 +1,137 @@
+package Duke.ui;
+
+import Duke.task.Task;
+import Duke.tasklist.TaskList;
+
+import java.util.List;
+import java.util.Scanner;
+
+import static Duke.constants.Constants.*;
+
+public class Ui {
+
+    private Scanner in;
+
+    /**
+     * Constructor takes in user input
+     */
+    public Ui() {
+        in = new Scanner(System.in);
+    }
+
+    /**
+     * Prints "Your task?"
+     *
+     * @return the line read in after trimming trailing and leading spaces
```

Comment:
I like how you write the java documentation, it makes us understand your code clear. 👍